supabase/migrations/20240712060620_service_requests.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE ONLY "public"."service_requests" ADD COLUMN "urgent" boolean DEFAULT FALSE NOT NULL;
```

Comment:
This migration has already been pushed to the shared supabase instance. In the future we should be doing this automatically through GitHub actions

Reply:
is the github action in place for this? I just followed docs for how to do migrations

Reply:
It should be a part of a CD pipeline in my opinion. We should make sure that changes to prod only happen through one CD pipeline (using Github actions), and if you need to do something outside the pipeline, then you'd want to write up a doc which explains the manual change you are making